### PR TITLE
Update/bid tracker remaining bids message

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Will discuss at Retro. Not sure if this is what was agreed upon.

### CODE REVIEW

File: src/Components/BidTracker/BidTracker.jsx [hunk 1]
```diff
@@ -62,16 +62,16 @@ class BidTracker extends Component {
   render() {
```
Comment (mjoyce91) [inline src/Components/BidTracker/BidTracker.jsx]: I'm trying to tell - was anything actually updated in this file, or did the line spacing just change?

File: src/Components/BidTracker/OverlayAlert/ClosedAlert/ClosedAlert.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import LinkButton from '../../../LinkButton';
 const ClosedAlert = ({ title, date, id }, { condensedView }) => (
   <div className="bid-tracker-alert-container bid-tracker-alert-container--closed">
     <div className="top-text">
-      {title} has been filled
+        Your bid  on {title} is no longer applicable.
```
Comment (mjoyce91) [inline src/Components/BidTracker/OverlayAlert/ClosedAlert/ClosedAlert.jsx]: 👍

File: src/Components/BidTracker/BidTracker.jsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidTracker.jsx]: In the end, after being further explained the intention of the ticket, no. Before, when I added a whole alert and was counting approved bids and non-approved, yes. I guess when I removed the unnecessary code, the spacing updated.

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidTracker.jsx]: image

Comment (mjoyce91) [reply src/Components/BidTracker/BidTracker.jsx]: Cool, see if you can revert the changes so that the spacing is back to normal

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidTracker.jsx]: ok, i'll try

**Pull request merged**